algorithms/unix/path/split.py

```diff
@@ -0,0 +1,25 @@
+"""
+Splitting a path into 2 parts
+Example:
+Input: https://algorithms/unix/test.py   (for url)
+Output:
+    part[0]: https://algorithms/unix
+    part[1]: test.py
+
+Input: algorithms/unix/test.py          (for file path)
+Output:
+    part[0]: algorithms/unix
+    part[1]: test.py
+"""
+import os
+
+def split(path):
+    parts = []
+    split_part = path.split('/')
+    # Remove the last element in split_part
+    split_part.pop(-1)
+    # Takt the origin path without the last part
+    parts.append("/".join(split_part))
+    # Take the last element of list
+    parts.append(path.split('/')[-1])
```

Comment:
maybe using `path.rpartition('/')` can be easier ? [`str.rpartition`](https://docs.python.org/3/library/stdtypes.html#str.rpartition).

Reply:
This function `str.rpartition` is better